src/main/java/edu/cornell/kfs/fp/batch/service/impl/AccountingDocumentGeneratorBase.java

```diff
@@ -111,6 +124,38 @@ protected Note buildDocumentNote(AccountingXmlDocumentNote xmlNote) {
         note.setNotePostedTimestampToCurrent();
         return note;
     }
+    
+    protected void populateDocumentAttachments(T document, AccountingXmlDocumentEntry documentEntry) {
+        documentEntry.getBackupLinks().stream()
+            .filter(link -> StringUtils.isNotBlank(link.getLinkUrl()))
+            .map(link -> buildDocumentNoteAttachment(document, link))
+            .forEach(document :: addNote);

```

Comment:
I would suggest removing the spaces around the double-colon, for consistency with how our other method references are formatted.

Reply:
can do